postprocessing/comstockpostproc/cbecs.py

```diff
@@ -66,9 +66,8 @@ def __init__(self, cbecs_year, truth_data_version, color_hex=NamingMixin.COLOR_C
             self.add_comstock_building_type_column()
             self.add_vintage_column()
             self.add_energy_intensity_columns()
-
-        # Calculate weighted area and energy consumption columns
-        self.add_weighted_area_and_energy_columns()
+            # Calculate weighted area and energy consumption columns
```

Comment:
I guess adding the weighted area and energy columns is redundant if you reload from a CSV that already has them, but not redundant if you reload from a CSV that doesn't have them. The redundancy is probably better than assuming they already exist.